src/test/java/core/basesyntax/strategy/impl/TransactionHandlerTest.java

```diff
@@ -0,0 +1,137 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.impl.StorageDaoImpl;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.storage.Storage;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TransactionHandlerTest {
```

Comment:
There should be a separate test for each class. You cannot test more than one class in one test